ReSwiftTests/TypeHelperTests.swift

```diff
@@ -22,28 +22,32 @@ class TypeHelper: QuickSpec {
 
             it("calls methods if the source type can be casted into the function signature type") {
                 var called = false
-                let reducerFunction: (AppState1, Action) -> AppState1 = { state, action in
+                let reducerFunction: (Action, AppState1) -> AppState1 = { action, state in
                     called = true
 
                     return state
                 }
 
-                withSpecificTypes(AppState1(), action: StandardAction(""),
-                    function: reducerFunction)
+                if let specificStateType = castToExpectedType(AppState1(), action: StandardAction(""),
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 102 characters (line_length)